reframe/core/pipeline.py

```diff
@@ -1239,6 +1239,9 @@ def compile(self):
             self.build_system.srcfile = self.sourcepath
             self.build_system.executable = self.executable
 
+        if type(self.build_system).__name__ == 'EasyBuild':
+            self.build_system._installpath = os.path.join(self._stagedir, 'easybuild')
```

Comment:
If you just want to put build/install/sources in the same directory, just use `eb --prefix`?

Reply:
I'm trying `eb --prefix` on Daint but as it is configured, it still gets the build/install/sources from the environment variables. For instance, 
```
eb zlib-1.2.11.eb --prefix=$HOME/easybuild
```
creates `$HOME/easybuild` with only `ebfiles_repo/zlib/zlib-1.2.11.eb` inside.